conf/messages

```diff
@@ -3775,8 +3775,8 @@ taxDue.trustDetails.label=Help for trusts for vulnerable beneficiaries
 taxDue.trustDetails.content=Trusts for vulnerable beneficiaries get special tax treatment. The workings out on this page do not apply to this type of trust. Find out how to <a href="{0}" target="_blank">work out the amount of Capital Gains Tax due (opens in a new window)</a> for this type of trust.
 taxDue.error.required=Enter the amount of Capital Gains Tax due for this return
 taxDue.error.invalid=Amount of Capital Gains Tax due must be an amount of money, for example 493 or 2,040.59
-taxDue.error.tooSmall=Amount of Capital Gains Tax due must be an amount of money between 0 and 50,000,000,000
-taxDue.error.tooLarge=Amount of Capital Gains Tax due must be an amount of money between 0 and 50,000,000,000
+taxDue.error.tooSmall=Amount of Capital Gains Tax due must be an amount of money, between 0 and 50,000,000,000
```

Comment:
could we double check with Jean to see if this comma is correct? On the other similar messages it's not there

Reply:
Similar pattern is followed on: taxDue.error.invalid